frontend_build/src/parse_bundle_plugin.js

```diff
@@ -72,7 +73,8 @@ var parseBundlePlugin = function(data, base_dir) {
       __kolibriModuleName: JSON.stringify(data.name),
       __events: JSON.stringify(data.events || {}),
       __once: JSON.stringify(data.once || {})
-    })
+    }),
+    new extract$trs(path.relative(base_dir, path.dirname(data.stats_file)), data.name)
```

Comment:
Yes, all Vue file strings will be extracted during JS compilation.

I did not notice a big slowdown, and I only parse the AST for vue files (however, I am parsing the AST for all Vue files already that exist, and I didn't notice any slowdown).

It is also done asynchronously, and only once the bundle itself has already been emitted, so it shouldn't interfere with the build process too much.